Project.toml

```diff
@@ -0,0 +1,9 @@
+name = "DifferenceEquations"
+uuid = "e0ca9c66-1f9e-11ec-127a-1304ce62169c"
+authors = ["Jesse Perla <jesseperla@gmail.com> and contributors"]
+version = "0.1.0"
+
+[deps]
```

Comment:
You should add a [compat] section as well if you want to register it :wink:

Reply:
Ah damn. Thank you for catching that! Will do.

Reply:
Get rid of my name on that :-)  or at least lets not suggest that I get most of the credit.  Lets use the sciml standard for the authors

Reply:
Sorry for being unclear.  I just meant "put your name on it" and then we can revisit later depending on how chris organizes this stuff.